container/sagemaker/tensorflow-serving.js

```diff
@@ -37,6 +56,10 @@ function ping(r) {
     r.subrequest(uri, options, callback)
 }
 
+function remove_tfs_method(uri) {
```

Comment:
I don't understand the objective of this function. Is it possible to document it?

Reply:
i'll add a comment

Reply:
removed this function